src/racf_kdfaes_fmt_plug.c

```diff
@@ -38,7 +38,7 @@ john_register_one(&fmt_racf_kdfaes);
 #define FORMAT_NAME             ""
 #define FORMAT_TAG              "$racf$*"
 #define FORMAT_TAG_LEN          (sizeof(FORMAT_TAG)-1)
-#define ALGORITHM_NAME          "KDFAES (DES + HMAC-SHA256/" ARCH_BITS_STR " + AES-256)"
+#define ALGORITHM_NAME          "DES + HMAC-SHA256/" ARCH_BITS_STR " + AES-256"
```

Comment:
IIRC, `KDFAES` is an official name for their KDF, so I'd keep it in here. It's similar to how we mention `PBKDF2` or `bcrypt` and not just their underlying primitives.

Reply:
reverted